src/primitives/circle.jl

```diff
@@ -5,24 +5,54 @@
 """
     Circle(plane, radius)
 
-A circle on a `plane` with given `radius`.
+A circle on a `plane` with given `center` and `radius`.
 
 See also [`Disk`](@ref).
 """
 struct Circle{T} <: Primitive{3,T}
   plane::Plane{T}
+  center::Point{3,T}
```

Comment:
Should this center be expressed in terms of parametric coordinates (u,v)? I find it a bit strange that we need to provide a 3D center after choosing a Plane of reference. We only have 2 degrees of freedom to choose the center after the Plane is given.